samples/diagnostics/program-visualization/README.md

```diff
@@ -0,0 +1,22 @@
+# Visualization of Q# Programs #
+
+This sample demonstrates the use of the `%trace` magic command in IQ# to visualize your Q# program within a Jupyter notebook.
+
+<p align="center">
```

Comment:
If possible, it may be good to switch to using Markdown image syntax, even though that means you won't be able to center it. That way, your image will do the right thing when imported into the samples browser as well.

Reply:
Ah ok, will do! Another reason I was using html was because the image size was way too big and I couldn't get it to be resized in Markdown.